src/recensio/plone/browser/person.py

```diff
@@ -0,0 +1,19 @@
+from plone import api
+from Products.Five import BrowserView
+
+
+class BaseView(BrowserView):
+    def __call__(self):
+        """This is needed for the migrated Person instances.
```

Comment:
```suggestion
        """TODO
        This is needed for the migrated Person instances.
```
Add a TODO note, otherwise we forget about cleaning this up in the future.